### ISSUE TITLE
what is the baseline for your md <RL/docs/guides/muon-optimizer.md>

### ISSUE THREAD

**Issue by wangyuwen1999** [NVIDIA-NeMo/RL#3567]
**The value change of muon_extra_scale_factor in post-training**
Recently, I try to use muon opt in my experiments, then I try to get the correct hyperparameters for my muon optimizer. I try to get these  hyperparameters from the existed experiment, and I find that your experiment in muon for post-training in muon-optimizer.md use muon_extra_scale_factor = 0.5 rather then muon_extra_scale_factor = 0.2 for SFT. I am confuse about your change for muon_extra_scale_factor, does it means muon need muon_extra_scale_factor to be larger in post-training then in SFT?

**what I need**
+ what is the baseline of post-training in muon-optimizer.md, please give me the launch scripts or commands.
+ why changed muon_extra_scale_factor from 0.2 to 0.5. Does it make sense?

**Comment by wangyuwen1999** [comment]
I also can not replay your RL result on RL/docs/guides/muon-optimizer.md.

**Comment by wangyuwen1999** [comment]
because the reward of response is all zero by Qwen 2.5 - 7B。

**Comment by wangyuwen1999** [comment]
I am so doubt about your result.
